src/App.tsx

```diff
@@ -1,61 +1,120 @@
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 import './App.scss';
+import { Todo } from './components/types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const getUserById = (userId: number) => {
+  return usersFromServer.find((user) => user.id === userId) || null;
+};
+
+const newTodo: Todo[] = todosFromServer.map((todo) => ({
```

Comment:
```suggestion
const todosWithUser: Todo[] = todosFromServer.map((todo) => ({
```
Actually, it's an array of todos. The name should be clear